src/App/Repository/SupplierRepository.php

```diff
@@ -189,13 +189,19 @@ protected function bindValues($databaseBindings, $query, Supplier $supplier)
      * @param int $page
      * @return mixed
      */
-    public function findLotSuppliers($lotIds, $paginate = false, $limit = 20, $page = 0)
+    public function findLotSuppliers($lotIds, $paginate = false, $limit = 20, $page = 0, $unique = false)
     {
-
-        $sql = 'SELECT DISTINCT s.id, s.salesforce_id, s.name, ls.trading_name, s.phone_number, s.street, s.city, s.postcode, s.website, s.country, ls.contact_name, ls.contact_email, ls.trading_name, IFNULL(ls.trading_name, s.name) as order_name FROM `ccs_suppliers` s
+        if ($unique) {
+            $sql = 'SELECT DISTINCT s.id, s.salesforce_id, s.name, ls.trading_name, s.phone_number, s.street, s.city, s.postcode, s.website, s.country, ls.trading_name, IFNULL(ls.trading_name, s.name) as order_name FROM `ccs_suppliers` s
 JOIN `ccs_lot_supplier` ls ON ls.supplier_id=s.salesforce_id
 WHERE ls.lot_id IN (\'' . $lotIds . '\')
 ORDER BY order_name';
+        } else {
+            $sql = 'SELECT DISTINCT s.id, s.salesforce_id, s.name, ls.trading_name, s.phone_number, s.street, s.city, s.postcode, s.website, s.country, ls.contact_name, ls.contact_email, ls.trading_name, IFNULL(ls.trading_name, s.name) as order_name FROM `ccs_suppliers` s
```

Comment:
Because this method is used somewhere else and I didnt want to change the behaviour for the others so I have added a condition param which is the "unique" parameter. 

For my usage, I don't need the contact_name and contact_email because one supplier can have more than one contact_name or contact_email. But I do need this list to be unique for  suppliers only. By removing the fields that I don't need and adding DISTINCT to the sql, I am able to retrieve what I need.